src/sidebar/helpers/view-filter.ts

```diff
@@ -182,5 +168,5 @@ export function filterAnnotations(annotations, filters) {
     .filter(ann => {
       return ann.id && rootFilter.matches(ann);
     })
-    .map(ann => /** @type {string} */ (ann.id));
+    .map(ann => ann.id as string);
```

Comment:
```suggestion
    .map(ann => ann.id!);
```

I think this is non-null cast? If so, we can use `!` to make that more obvious.